### ISSUE TITLE
Incentives section in the book

### ISSUE THREAD

**Issue by upavloff** [smartcontracts/eth2-book#34]
I might be mistaken, but I don't see a section about rewards and penalties in the book. I think it would be nice to have one. Nice work otherwise :)

**Comment by smartcontracts** [comment]
Yeah... this book has been stalled for a long time. I intend to get back to it once things are in a more stable place for Optimism and the Collective.
